cmd/promtool/unittest.go

```diff
@@ -145,6 +145,9 @@ func resolveAndGlobFilepaths(baseDir string, utf *unitTestFile) error {
 		if err != nil {
 			return err
 		}
+		if len(m) <= 0 {
```

Comment:
I don't think returning an error in this case would be right as it is not necessary to always find a file for the pattern. A warning log would be better instead.

Reply:
returning an error forces the developer to check the file path or the glob pattern again since there was no match. just making a warning log might eventually get ignored. A developer would mostly want to match a specific file(s) with the glob pattern or file path.

Reply:
I still feel we should not return an error here. Thoughts @brian-brazil?